echopype/utils/prov.py

```diff
@@ -1,5 +1,5 @@
 from datetime import datetime as dt
-from pathlib import PosixPath
+from pathlib import Path
```

Comment:
Oh wow, didn't realize that this module was locked into `PosixPath`! Nice fix @emiliom! Do you know if this is the only place that `PosixPath` is found?? I hope so!

Reply:
> Do you know if this is the only place that PosixPath is found??

It looks that way.

> Oh wow, didn't realize that this module was locked into PosixPath!

My bad :disappointed: . I think I introduced that.